advice_for_future_giraf/PO_advice.md

```diff
@@ -0,0 +1,151 @@
+# PO advice for next year's PO group
+
+This document serves as a list of advice for the next PO group based on the experience we gathered throughout the semester.
+The PO group's main responsibility is to communicate with the customers of the GIRAF project and document this interaction.
+It is important that the customers' reactions, opinions and wishes for the program are written down, as these are the foundation for all the decisions that you, as the PO group, will take.
+In the following sections we provide an in-depth description of how we advice customer contact to be handled, and how the information gathered should be used.
+
+## Customer contact
+In the start of the semester Ulrik will most likely schedule some meetings with the customers which everyone should attend.
```

Comment:
Specify who 'everyone' is. Everyone in your group, everyone in the giraf project, or everyone in the entire world? :)